[2024-06-28T09:15:10.019+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T09:15:10.169+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.train_model manual__2024-06-28T09:15:00.759910+00:00 [queued]>
[2024-06-28T09:15:10.179+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.train_model manual__2024-06-28T09:15:00.759910+00:00 [queued]>
[2024-06-28T09:15:10.180+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T09:15:10.192+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): train_model> on 2024-06-28 09:15:00.759910+00:00
[2024-06-28T09:15:10.199+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=73) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T09:15:10.200+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'train_model', 'manual__2024-06-28T09:15:00.759910+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmporzywzx7']
[2024-06-28T09:15:10.200+0000] {standard_task_runner.py:63} INFO - Started process 87 to run task
[2024-06-28T09:15:10.201+0000] {standard_task_runner.py:91} INFO - Job 62: Subtask train_model
[2024-06-28T09:15:10.244+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.train_model manual__2024-06-28T09:15:00.759910+00:00 [running]> on host 72af1dab7ef8
[2024-06-28T09:15:10.332+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T09:15:00.759910+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T09:15:00.759910+00:00'
[2024-06-28T09:15:10.333+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T09:15:10.514+0000] {warnings.py:112} WARNING - /opt/***/scripts/train_model.py:16: FutureWarning: Series.fillna with 'method' is deprecated and will raise in a future version. Use obj.ffill() or obj.bfill() instead.
  data['temperature_2m'].fillna(method='ffill', inplace=True)

[2024-06-28T09:15:10.566+0000] {logging_mixin.py:188} INFO - Mean Squared Error: 7.210475408331276
[2024-06-28T09:15:10.567+0000] {logging_mixin.py:188} INFO - R-squared: 0.8558007380909088
[2024-06-28T09:15:10.571+0000] {logging_mixin.py:188} INFO - Model has been saved to ./temperature_prediction_model.pkl
[2024-06-28T09:15:10.572+0000] {python.py:237} INFO - Done. Returned value was: ./temperature_prediction_model.pkl
[2024-06-28T09:15:10.572+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T09:15:10.614+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=train_model, run_id=manual__2024-06-28T09:15:00.759910+00:00, execution_date=20240628T091500, start_date=20240628T091510, end_date=20240628T091510
[2024-06-28T09:15:10.656+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T09:15:10.680+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T09:15:10.684+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
